bbhash_iter_test.go

```diff
@@ -13,138 +13,70 @@ import (
 	"github.com/relab/bbhash/internal/fast"
 )
 
+// String taken from https://www.lipsum.com/
+const input string = "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
+const word string = "Hello"
```

Comment:
Missing blank line between these `const` declarations. This usually doesn't happen if you have the Go extension and gopls configured in VScode.

That said, not sure it is necessary to keep the `word` declaration here; it can be moved inside the Test func where it is needed.